pkgs/googleapis_firestore_v1/README.md

```diff
@@ -16,6 +16,37 @@ See also:
 
 This package is a generated gRPC client used to access the Firestore API.
 
+## Example
```

Comment:
Maybe add a link to the more complete `pkgs/googleapis_firestore_v1/example/example.dart`?